boa_engine/src/context/intrinsics.rs

```diff
@@ -764,6 +764,12 @@ pub struct IntrinsicObjects {
     /// [`%parseInt%`](https://tc39.es/ecma262/#sec-parseint-string-radix)
     parse_int: JsFunction,
 
+    /// [`%escape%`](https://tc39.es/ecma262/#sec-escape-string)
+    escape: JsFunction,
```

Comment:
Should this also be feature-gated?

Reply:
Haha, we commented the same thing :laughing: